man/bundle-add.ronn

```diff
@@ -0,0 +1,27 @@
+bundle-add(1) -- Add gem to Gemfile and run bundle install
+================================================================
+
+## SYNOPSIS
+
+`bundle add` [--group=GROUP] [--version=VERSION] [--source=SOURCE]  [GEM]
+
+## DESCRIPTION
+Adds the named gem to Gemfile and run `bundle install`.
+
+Example:
+
```

Comment:
There needs be an example without any options,  ie: `bundle add rails` (this should be the first example)